_includes/tutorials/finding-distinct/kstreams/markup/dev/make-topology.adoc

```diff
@@ -1,6 +1,9 @@
 Then create the following file at `src/main/java/io/confluent/developer/FindDistinctEvents.java`
 
-Focusing on the `buildTopology` method, note how the Kafka Streams topology relies on a `https://docs.confluent.io/current/streams/javadocs/org/apache/kafka/streams/kstream/Transformer.html[Transformer]` and a `https://docs.confluent.io/current/streams/javadocs/org/apache/kafka/streams/state/WindowStore.html[Window Store]` to filter out the duplicate IP addresses.   Events are de-duped within a 2 minute window, and unique clicks are produced to a new topic named `distinct-clicks`.
+Focusing on the `buildTopology` method, note how the Kafka Streams topology relies on a `https://docs.confluent.io/current/streams/javadocs/org/apache/kafka/streams/kstream/ValueTransformerWithKey.html[ValueTransformerWithKey]` and a `https://docs.confluent.io/current/streams/javadocs/org/apache/kafka/streams/state/WindowStore.html[Window Store]` to filter out the duplicate IP addresses.   Events are de-duped within a 2 minute window, and unique clicks are produced to a new topic named `distinct-clicks`.
+
+NOTE: Note that we are using `https://docs.confluent.io/current/streams/javadocs/org/apache/kafka/streams/kstream/ValueTransformerWithKey.html[ValueTransformerWithKey]` here instead of `https://docs.confluent.io/current/streams/javadocs/org/apache/kafka/streams/kstream/Transformer.html[Transformer]` since we need keys to transform data, but there is no need to re-key the stream. 
```

Comment:
Unfortunately, call-outs don't render correctly at the moment.  Can you remove the `NOTE:` text? maybe you could do something like `**_Note that we are using_**` instead.  We working on adding support for call-outs.

Reply:
Oh I see! I got used to Asciidoctor in my own projects, that's why I'm using its features without thinking about render support :-)